templates/docs/patterns/newsletter-signup/index.md

```diff
@@ -0,0 +1,316 @@
+
+## 4-col
+
+This variant is also used within a grid, and takes up 4 columns on all dimensions.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/newsletter-signup/4-col" class="js-example" data-lang="jinja">
+View example of the 4-col newsletter signup
+</a></div>
+
+## Jinja Macro
+
+The `vf_newsletter_signup` Jinja macro can be used to generate a Newsletter signup pattern. The API for the macro is shown below.
+
+### Parameters
+
+<div style="overflow: auto;">
+  <table>
+    <thead>
+      <tr>
+        <th style="width: 220px;">Name</th>
+        <th style="width: 160px;">Required?</th>
+        <th style="width: 160px;">Type</th>
+        <th style="width: 160px;">Default</th>
+        <th style="width: 250px;">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <code>title_text</code>
+        </td>
+        <td>
+          Yes
+        </td>
+        <td>
+          <code>string</code>
+        </td>
+        <td>
+          <code>N/A</code>
+        </td>
+        <td>
+          <code>h2/h3</code> title text<br/>
+          <code>h2</code> for <code>`25-75`, `50-50` variants</code><br/>
+          <code>h3</code> for <code>`2-col`, `4-col` variants</code><br/>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>input_label</code>
+        </td>
+        <td>
+          Yes
+        </td>
+        <td>
+          <code>string</code>
+        </td>
+        <td>
+          <code>N/A</code>
+        </td>
+        <td>
+          Label for the input field
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>checkbox_id</code>
+        </td>
+        <td>
+          No
+        </td>
+        <td>
+          <code>string</code>
+        </td>
+        <td>
+          <code>canonicalUpdatesOptIn</code>
+        </td>
+        <td>
+          Label for the input field
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>checkbox_label</code>
+        </td>
+        <td>
+          Yes
+        </td>
+        <td>
+          <code>string</code>
+        </td>
+        <td>
+          <code>N/A</code>
+        </td>
+        <td>
+          Label for the checkbox
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>layout</code>
+        </td>
+        <td>
+          No
+        </td>
+        <td>
+          <code>string</code>
+        </td>
+        <td>
+          <code>25-75</code>
+        </td>
+        <td>
+          Variant of the newsletter signup pattern<br/>
+          One of <code>`25-75`</code>, <code>`50-50`</code>, <code>`2-col`</code>, <code>`4-col`</code>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>form_id</code>
+        </td>
+        <td>
+          Yes
+        </td>
+        <td>
+          <code>string</code>
+        </td>
+        <td>
+          <code>N/A</code>
+        </td>
+        <td>
+          Marketo form id. For example <code>mktoForm_XXXX</code>
```

Comment:
This is still marketo specific, you should update is so it reads something like "the ID that will be passed to the form element"